models/user/user.go

```diff
@@ -501,19 +501,19 @@ func GetUserSalt() (string, error) {
 // Note: The set of characters here can safely expand without a breaking change,
 // but characters removed from this set can cause user account linking to break
 var (
-	customCharsReplacement    = strings.NewReplacer("Æ", "AE")
-	removeCharsRE             = regexp.MustCompile(`['´\x60]`)
```

Comment:
This is mentioned in docs, should we also change the doc? 
ps: actually I don't know what it is used for.
![image](https://github.com/go-gitea/gitea/assets/18380374/bc331321-1a44-46e1-b9ee-883268b5751a)

Reply:
also in app.example.ini
![image](https://github.com/go-gitea/gitea/assets/18380374/6ca8892a-533f-4d4e-a775-efd4f5956471)

Reply:
They are the same. `\x60` is `` ` ``

![image](https://github.com/go-gitea/gitea/assets/2114189/26be9c1d-e0f3-4334-b290-6671e59059d3)